frontend/routes/viewer/components/compare/components/compareDiffItem/compareDiffItem.component.tsx

```diff
@@ -0,0 +1,97 @@
+/**
+ *  Copyright (C) 2017 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import * as React from 'react';
+import * as dayjs from 'dayjs';
+
+import { ArrowsAltH } from '../../../../../components/fontAwesomeIcon';
+import Checkbox from '@material-ui/core/Checkbox';
+import { RevisionsSelect } from '../revisionsSelect/revisionsSelect.component';
+import { Container, ModelData, Name, Revisions, CurrentRevision, CompareIconWrapper } from './compareDiffItem.styles';
+import { renderWhenTrue } from '../../../../../../helpers/rendering';
+
+interface IProps {
+	className?: string;
+	name: string;
+	baseRevision: any;
+	currentRevision: any;
+	targetDiffRevision: any;
+	revisions: any[];
+	selected?: boolean;
+	onSelectionChange: (event, selected) => void;
+	onRevisionChange: (modelProps) => void;
+}
+
+export class CompareDiffItem extends React.PureComponent<IProps, any> {
+	public static defaultProps = {
+		selected: false
+	};
+
+	public get currentRevisionName() {
+		return this.props.baseRevision.tag || dayjs(this.props.baseRevision.timestamp).format('DD/MM/YYYY');
```

Comment:
We have a lot of copies of `dayjs(...)`. Since we'd always want this to be consistent throughout the component, can we refactor this out to a utils function?

Reply:
Do you want the same format everywhere? Because I see different formats in the app:
- 15/05/2019 (Filter panel, Compare)
- 15 May 2019 (Risks, Issues, Groups)
- 08:30 15 May (Groups details form)

Reply:
just consistent throughout the particular component is fine! :)

Reply:
Done :)

Reply:
Hi @sanmont3drepo  This file was outdated. I moved dayjs function to services. Please check again, and let me know if it's ok for you :)

Reply:
@dleszcz  the code has been changed from 
`dayjs(child.date).format('DD/MM/YYYY');` to `formatDate(child.date, 'DD/MM/YYYY');`
What @carmenfan  meant was to change the `dayjs(child.date).format('DD/MM/YYYY');`  to somehing like  `formatShortDate(child.date);`

where `formatShortDate` is defined like 
`export const formatShortDate = (date) => dayjs(date).format('DD/MM/YYYY');`

Reply:
done